Lombiq.HelpfulExtensions/Extensions/OrchardRecipeMigration/Services/OrchardExportToRecipeConverter.cs

```diff
@@ -67,15 +78,18 @@ await _contentConverters
             }
         }
 
-        foreach (var converter in _exportConverters)
-        {
-            await converter.UpdateContentItemsAsync(export, contentItems);
-        }
+        var urlHelper = _urlHelperFactory.GetUrlHelper(_actionContextAccessor.ActionContext!);
 
-        var recipe = JObject.FromObject(new RecipeDescriptor())!;
-        recipe["steps"] = new JsonArray(JObject.FromObject(new { name = "content", data = contentItems }));
+        bool hasNextPage = page < totalPages;
+        string nextUrl = hasNextPage ? urlHelper.Action(nameof(Convert), "OrchardRecipeMigrationAdmin", new { page = page + 1 }) : null;
 
-        return recipe.ToString();
+        return new
+        {
+            Processed = page * pageSize > totalItems ? totalItems : page * pageSize,
+            Total = totalItems,
+            NextPage = nextUrl,
+            ContentItems = contentItems,
+        };
```

Comment:
Couldn't this be a model? Instead of 
![image](https://github.com/user-attachments/assets/feefa013-3c92-4940-ae68-402a7f78339f)

And I think then you can do `public async Task<ModelName> ConvertAsync`